File: dotnet/Multiplayer/Matchmaking/MatchmakingLeaveResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Matchmaking
{
    /// <summary>
    /// Response returned when a player successfully leaves a matchmaking lobby.
    /// </summary>
    public class MatchmakingLeaveResponse : ApiResponse<EMatchmakingLeaveError>
    {
        /// <summary>
        /// Confirmation message for leaving the lobby.
        /// </summary>
        public string Message { get; set; } = string.Empty;
    }
}

File: java/src/main/java/com/michitai/multiplayer/matchmaking/MatchmakingLeaveResponse.java
package com.michitai.multiplayer.matchmaking;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response returned when a player leaves a matchmaking lobby.
 */
public class MatchmakingLeaveResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
